ext/gd/tests/bug39366.phpt

```diff
@@ -4,7 +4,7 @@ Bug #39366 (imagerotate does not respect alpha with angles>45)
 gd
 --SKIPIF--
 <?php
-    if (!function_exists("imagerotate")) die("skip requires bundled GD library\n");
+    if (!function_exists("imagerotate")) die("skip requires imagerotate function");
```

Comment:
Does it _just_ require this function?

Reply:
Well, obviously not. But instead of adding more checks, maybe we need fewer. Can this fail at all?

My inspiration for `skip requires imagerotate function` was a combination of the test description and some other tests that do the same thing. But I think these may all date back to a time when the bundled libgd had `gdImageRotateInterpolated()` but the external libgd might not have. These days both should have it (it's ten years old, and not guarded by an `#ifdef`), so the `function_exists()` check is probably pointless. None of the other `image*` functions are `#ifdef`ed either.

I can remove the check if you think that's best.

Reply:
Removing them is probably best then yes.

Reply:
Easy enough. I tracked down the exact commit where the `#ifdef` was removed for `imagerotate()`, it was even older than I thought.